10_SL1/errors.yaml

```diff
@@ -181,6 +204,23 @@ Errors:
   id: "UV_TEMP_SENSOR_FAILED"
   approved: true
 
+- code: "10210"
+  title: "TEMPERATURE SENSOR FAILED"
+  text: "The %(sensor__map_HardwareDeviceId)s sensor failed. Check the wiring and connection."
+  id: "TEMP_SENSOR_FAILED_ID"
+  approved: true
+
+- code: "10211"
+  title: "TEMPERATURE OUT OF RANGE"
+  text: "%(sensor__map_HardwareDeviceId)s not in range!
```

Comment:
Nebylo py lespi to preformulovat, aby byl nazev senzoru uprostred vety a mohli tam zustat maly pismena?